baybe/constraints/discrete.py

```diff
@@ -78,6 +107,22 @@ def get_invalid(self, data: pd.DataFrame) -> pd.Index:  # noqa: D102
 
         return data.index[mask_bad]
 
+    def to_polars(self) -> pl.Expr:
+        """Translate the constraint to Polars expression for filtering.
+
+        Returns:
+            The Polars Expr object to pass as an argument to filter().
+
+        """
+        op = _threshold_operators[self.condition.operator]
+        # Identify columns present in both self.parameters and df.columns
+
+        # Get the product of columns using reduce function and with an alias "prod"
+        expr = pl.reduce(lambda acc, x: acc * x, pl.col(self.parameters)).alias("prod")
```

Comment:
Perhaps stupid question, but why do we need to place an alias if we never access it via the given name?

Reply:
wasnt needed, removed